notifications_utils/formatters.py

```diff
@@ -619,8 +623,45 @@ def add_language_divs(_content: str) -> str:
 def remove_language_divs(_content: str) -> str:
     """Remove the tags from content. This fn is for use in the email
     preheader, since this is plain text not html"""
-    content = re.compile(FR_OPEN).sub("", _content)
-    content = re.compile(FR_CLOSE).sub("", content)
-    content = re.compile(EN_OPEN).sub("", content)
-    content = re.compile(EN_CLOSE).sub("", content)
+    return remove_tags(_content, FR_OPEN, FR_CLOSE, EN_OPEN, EN_CLOSE)
+
+
+def add_ga_seal(_content: str) -> str:
+    """
+    Custom parser to add Global Affairs seal logo.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
+    """
+    ga_seal_regex = re.compile(f"{IMG_GLOBAL_AFFAIRS}")  # matches [[ga-seal]]
+    content = ga_seal_regex.sub(
+        r"""<div style="margin: 20px auto 30px auto;">
+          <img
+            src="https://assets.notification.canada.ca/gc-ga-seal.png"
+            alt="Global Affairs Canada / Affaires mondiales Canada"
+            height="339"
+            width="322"
+          />
+        </div>""",
+        _content,
+    )
+
+    return content
+
+
+def remove_ga_seal(_content: str):
+    return remove_tags(_content, IMG_GLOBAL_AFFAIRS)
```

Comment:
If I'm understanding correctly, this is used to remove the image from plain text emails? If so, should we instead provide a link to the image that can be copied / pasted in cause the recipient does not have an HTML email client? Would at least allow them to access it without an HTML email.

Reply:
I removed this code because it was superfluous. It was meant to remove the special `[[ircc-ga-seal]]` tags (not the html images per se) but it is unnecessary because we used the `sub` function of the `re` library which should had the tags replaced previously.

The concerns around plain text is valid. We are considering at the moment what would be the best format. From our talk with IRCC (for which these emergency changes are necessary), they need an email that looks the most with the template letter that they shared with the Talibans to let the people go through the flights. So plain text would properly not be great... and HTML has limitations, all based on the email client, unfortunately.

We are thinking of an iterative feature to follow through with transforming the HTML into PDF and attach it on top of these changes. Then, again, email client might be an issue too.

What is your take on this?

Reply:
My take would be, do all of the things we can. I'm assuming the emails you send are HTML with a text fallback? So, the HTML is the full one with the images. The text fallback has the links to the images so at least they could be opened if needed (not great, but better then not having them I'd think).

Having an attached PDF would also be good (if possible), I don't actually know what a text based email client would do with an attachment, does it show up?

Reply:
Without seeing the actual email template, it's hard to tell what would work, but possibly doing something like replacing `[[ircc-ga-seal]]` with the alt text and link in the text version? `Global Affairs Canada / Affaires mondiales Canada: https://assets.notification.canada.ca/gc-ircc-ga-seal.png`. Similar with the armory link?

Or, maybe you don't even need the link in that case as you have the alt text?